### ISSUE TITLE
ServerOptions numChannels=2 gives 8 channels

### ISSUE THREAD

**Issue by jpburstrom** [supercollider/supercollider#4250]
Environment
-----------

* Your SuperCollider version: 3.10.1-rc1
* Your operating system and version: MacOS 10.13.6

Steps to reproduce (for bugs)
-----------------------------

When rendering a file, setting numOutputBusChannels to 2 results in a 8-channel file.

```supercollider
(
var score = Score.new;
var options = ServerOptions.new.numOutputBusChannels = 2;
score.recordNRT("/tmp/test.osc", "/tmp/test.aiff", options:options, action: {
	SoundFile.use("/tmp/test.aiff", { |f| f.numChannels.debug("Channels") }
	);
})
)
```
This didn't happen in 3.10.

Expected Behavior
-----------------

The file should have 2 channels.  

Current Behavior
----------------

The output is 8 channels. It only happens when you set ```numOutputBusChannels``` to 2. I guess its due to a difference in default number of channels in the client and server. See ```SCClassLibrary/Common/Control/Server.sc``` and ```include/server/SC_WorldOptions.h```. 

A fix would be either to change the default channels to 2 in ```SC_WorldOptions.h```, or make ServerOptions always output ```num[In|Out]putBusChannels``` in ```-asOptionsString```.

**Comment by patrickdupuis** [comment]
Thanks for the bug report! See #4251 for a fix.

**Comment by patrickdupuis** [comment]
Fixed in #4251
